src/overview/components/Overview.jsx

```diff
@@ -14,6 +14,21 @@ class Overview extends React.Component {
         if (this.props.grabFocusOnMount) {
             this.refs['inputQuery'].focus()
         }
+
+        // Add an onscroll event listener to listen for scrolling events
+        window.addEventListener('scroll', (e) => {
```

Comment:
is this listener removed on component unmount? why not using react waypoint instead of worrying about this?